components/aws/sagemaker/tests/unit_tests/Dockerfile

```diff
@@ -13,8 +13,6 @@ RUN pip3 install -r requirements.txt
 COPY ./components/aws/sagemaker/dev_requirements.txt .
 RUN pip3 install -r dev_requirements.txt
 
-COPY . /app/
+ENTRYPOINT [ "bash" ]
 
-WORKDIR /app/components/aws/sagemaker/tests/unit_tests/
-
-ENTRYPOINT [ "bash", "./run_automated_test.sh" ]
\ No newline at end of file
+CMD [ "" ]
```

Comment:
Likewise to integration tests, might need to keep this the same for the sake of CI

Reply:
Let me know if CI fails, and I will revert these